### PR TITLE
Fix 404 links on API Platform's Philosophy

### PR DESCRIPTION
**Pull request by mheki**
Fixes links which currently lead to 404 pages. 
Replaces Angular js link with the link to Client Generator docs.

### CODE REVIEW

File: extra/philosophy.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ a full-stack all-in-one framework and a set of independent PHP components and bu
 The architecture promoted by the framework will distrust many of you but read this tutorial until the end and you will
 see how API Platform make modern development easy and fun again:
 
-* [Start by **creating a hypermedia REST API**](distribution/index.md) exposing structured data that can
+* [Start by **creating a hypermedia REST API**](/distribution/index.md) exposing structured data that can
```
Comment (sroze) [inline extra/philosophy.md]: `../distribution/index.md` would be better.

File: extra/philosophy.md [hunk 2]
```diff
@@ -26,9 +26,9 @@ a full-stack all-in-one framework and a set of independent PHP components and bu
...
   be understood by any compliant client such your apps but also as search engines (JSON-LD with Schema.org vocabulary).
   This API is the central and unique entry point to access and modify data. It also encapsulates the whole business logic.
-* [Then **create as many clients as you want using frontend technologies you love**](core/angularjs-integration.md): an HTML5/Javascript
+* [Then **create as many clients as you want using frontend technologies you love**](/client-generator/index.md): an HTML5/Javascript
```
Comment (sroze) [inline extra/philosophy.md]: Same, `../client-generator/index.md` would be better

**Comment by mheki** [comment]
Changed to relative paths @sroze 👍

**Comment by sroze** [comment]
Thank you @mheki

**Pull request merged**